src/css/custom.css

```diff
@@ -0,0 +1,91 @@
+/**
+ * Any CSS included here will be global. The classic template
+ * bundles Infima by default. Infima is a CSS framework designed to
+ * work well for content-centric websites.
+ */
```

Comment:
Interesting that we need so much more custom CSS here [than in Experimenter docs](https://github.com/mozilla/experimenter-docs/blob/main/src/css/custom.css).

I did notice one thing while poking around in dark mode:
![image](https://user-images.githubusercontent.com/13018240/148427870-8f1b7e0f-17dc-4f4f-a4d0-780892f6f2e8.png)

I tried to get better selectors and cover most of what we use, I think this does the trick (maybe put this at the bottom next to other dark theme tweaks):
```
/* We have to use !important here because mermaid inlines styles */
[data-theme="dark"] [id^=mermaid] .messageText, [data-theme="dark"] [id^=mermaid] .loopText {
  fill: var(--ifm-font-color-base) !important;
  stroke: var(--ifm-font-color-base) !important;
}

[data-theme="dark"] [id^=mermaid] .loopText tspan{
  fill: inherit;
}

[data-theme="dark"] [id^=mermaid] line, [data-theme="dark"] [id^=mermaid] path {
  stroke: #9c9c9c !important;
}
```
![image](https://user-images.githubusercontent.com/13018240/148427839-278c8e18-c740-4f7a-b418-5bf714ce5ca1.png)

Reply:
I'm assuming Jody wrote it to make it prettier.  I didn't look at the CSS.  Thanks for the tweaks, I'll grab them in a future PR